frontend/app/src/composables/api/blockchain/accounts.ts

```diff
@@ -66,8 +66,8 @@ export function useBlockchainAccountsApi() {
     const url = !Array.isArray(pay)
       ? `/blockchains/${chain}/xpub`
       : `/blockchains/${chain}/accounts`;
-    const payload = (Array.isArray(pay)) ? { accounts: pay } : pay;
-    return performAsyncQuery(url, nonEmptyProperties(payload));
+    const payload = (Array.isArray(pay)) ? { accounts: pay } : { ...pay, ...pay.xpub };
```

Comment:
Same here there are changes in the `accounts` branch can you also check then and pull them instead?

Reply:
Updated